apps/qubit/modules/term/templates/indexSuccess.php

```diff
@@ -254,11 +254,12 @@
           <?php endif; ?>
 
         <div id="sort-header">
-          <?php echo get_partial('default/sortPicker',
-            array(
-              'options' => array(
-                'lastUpdated' => __('Most recent'),
-                'alphabetic' => __('Alphabetic')))) ?>
+          <?php echo get_partial('default/sortPicker', array(
+            'options' => array(
+              'lastUpdated' => __('Most recent'),
+              'alphabetic'  => __('Alphabetic'),
+              'identifier'  => __('Reference code'),
+              'date'        => __('Date')))) ?>
```

Comment:
I thought so at first, but this change applies to linked descriptions underneath the terms indexSuccess page, not to the terms themselves. See http://imgur.com/a/1dlJa for clarification.